tests/js/spec/components/forms/selectField.spec.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import {SelectField} from 'app/components/forms';
+
+describe('SelectField', function() {
+  describe('render()', function() {
+    it('renders without form context', function() {
+      let wrapper = mount(
+        <SelectField
+          options={[{label: 'a', value: 'a'}, {label: 'b', value: 'b'}]}
+          name="fieldName"
+          value="a"
+        />
+      );
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('renders with form context', function() {
```

Comment:
what does the form context do? could you add an assert or something here? (or just comment and help me understand)